pkg/terraform/stages/baremetal/stages.go

```diff
@@ -0,0 +1,13 @@
+package baremetal
+
+import (
+	"github.com/openshift/installer/pkg/terraform"
+	"github.com/openshift/installer/pkg/terraform/stages"
+)
+
+// PlatformStages are the stages to run to provision the infrastructure in
+// Bare Metal.
+var PlatformStages = []terraform.Stage{
+	stages.NewStage("baremetal", "bootstrap", stages.WithNormalDestroy()),
+	stages.NewStage("baremetal", "masters"),
```

Comment:
One of these is not like the others... (ie, gcp and aws deploys masters ("cluster") before bootstrap). Is there any reason not to follow this here?

Reply:
I've tried it with masters first and there are problems with ironic. I don't know enough about this so leaving it as-is for now. 

<pre>
level=error
level=error msg=Error: could not contact Ironic API: context deadline exceeded
level=error
level=error msg=  on ../../tmp/openshift-install-masters-318431193/main.tf line 13, in resource "ironic_node_v1" "openshift-master-host":
level=error msg=  13: resource "ironic_node_v1" "openshift-master-host" {
level=error
level=error
level=error
level=error msg=Error: could not contact Ironic API: timeout reached
level=error
level=error msg=  on ../../tmp/openshift-install-masters-318431193/main.tf line 13, in resource "ironic_node_v1" "openshift-master-host":
level=error msg=  13: resource "ironic_node_v1" "openshift-master-host" {
level=error
level=error
level=error
level=error msg=Error: could not contact Ironic API: timeout reached
level=error
level=error msg=  on ../../tmp/openshift-install-masters-318431193/main.tf line 13, in resource "ironic_node_v1" "openshift-master-host":
level=error msg=  13: resource "ironic_node_v1" "openshift-master-host" {
level=error
level=error
level=fatal msg=failed to fetch Cluster: failed to generate asset "Cluster": failed to create cluster: failed to apply Terraform: error(BaremetalIronicAPITimeout) from Infrastructure Provider: Unable to the reach provisioning service. This failure can be caused by incorrect network/proxy settings, inability to download the machine operating system images, or other misconfiguration. Please check access to the bootstrap host, and for any failing services. 
</pre>